src/ext.rs

```diff
@@ -667,8 +667,9 @@ impl<T: AsRawSocket> AsSock for T {
 cfg_if! {
     if #[cfg(any(target_os = "macos", target_os = "ios"))] {
         use libc::TCP_KEEPALIVE as KEEPALIVE_OPTION;
-    } else if #[cfg(any(target_os = "openbsd", target_os = "netbsd"))] {
-        use libc::SO_KEEPALIVE as KEEPALIVE_OPTION;
+    } else if #[cfg(target_os = "openbsd")] {
```

Comment:
This is dropping the `netbsd` case.  Is it safe to do so?

Reply:
Yes.  NetBSD understands `TCP_KEEPIDLE` ([tcp(4)](https://netbsd.gw.com/cgi-bin/man-cgi?tcp++NetBSD-9.0)), the `cfg(unix)` case just below this one applies to it correctly.